staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -554,16 +554,25 @@ type CreateOptions struct {
 	// +optional
 	FieldManager string `json:"fieldManager,omitempty" protobuf:"bytes,3,name=fieldManager"`
 
-	// fieldValidation determines how the server should respond to
-	// unknown/duplicate fields in the object in the request.
-	// Introduced as alpha in 1.23, older servers or servers with the
-	// `ServerSideFieldValidation` feature disabled will discard valid values
-	// specified in  this param and not perform any server side field validation.
-	// Valid values are:
-	// - Ignore: ignores unknown/duplicate fields.
-	// - Warn: responds with a warning for each
-	// unknown/duplicate field, but successfully serves the request.
-	// - Strict: fails the request on unknown/duplicate fields.
+	// fieldValidation instructs the server on how to handle
+	// objects in the request (POST/PUT/PATCH) containing unknown
+	// or duplicate fields, provided that the `ServerSideFieldValidation`
+	// feature gate is also enabled. Valid values are:
+	// - Ignore: This will ignore any unknown fields that are silently
+	// dropped from the object, and will ignore all but the last duplicate
+	// field that the decoder encounters. This is the default behavior
+	// prior to v1.23 and is the default behavior when the
+	// `ServerSideFieldValidation` feature gate is disabled.
+	// - Warn: This will send a warning via the standard warning response
+	// header for each unknown field that is dropped from the object, and
+	// for each duplicate field that is encountered. The request will
+	// still succeed if there are no other errors, and will only persist
+	// the last of any duplicate fields. This is the default when the
+	// `ServerSideFieldValidation` feature gate is enabled.
+	// - Strict: This iwll fail the request with a BadRequest error if
```

Comment:
```suggestion
	// - Strict: This will fail the request with a BadRequest error if
```

(applies everywhere)